### DISCUSSION TITLE
Publish signed checksums when releasing a new version?

### DISCUSSION THREAD

**Discussion by evehash** [romanz/electrs#828] [General]
I know it can be validated with git and the tagged version, but since it is also distributed in zip and tar.gz it would be nice if the signed checksums were also available.

Is it possible? Thank you.

**Comment by romanz** [comment]
zip & tar.gz files are released automatically by GitHub, so I suggest using the git tagged version (which is signed by https://romanzey.de/pgp.txt) instead - see https://raspibolt.org/guide/bitcoin/electrum-server.html#build-from-source-code for an example.

**Reply by evehash** [reply]
Thank you Romanz.
